src/dc_imex.rs

```diff
@@ -258,9 +258,6 @@ pub unsafe fn dc_continue_key_transfer(
     msg_id: u32,
     setup_code: *const libc::c_char,
 ) -> bool {
-    let mut success = false;
```

Comment:
could you turn the setup_code into a &str make this a (mostly or totally) safe function?